llvm/include/llvm/CASObjectFormats/CASObjectReader.h

```diff
@@ -55,6 +59,8 @@ struct CASBlock {
   uint64_t AlignmentOffset;
   Optional<StringRef> Content;
   CASSectionRef SectionRef;
+  llvm::cas::CASID ID;
+  CASBlock(cas::ObjectProxy obj) : ID(obj.getID()) {}
```

Comment:
You might as well write a constructor that initialize all the fields.

Reply:
I don't think this is resolved. The CASBlock constructor should initialize all the fields so you don't need to assign to each individual field after construction.